### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dawnyy**
hihi so like if u doing idv like what concepts are you looking for?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # lol hi
+priestess x eli? coodinator x naib?
```
Comment (Schrodinger-09) [inline README.md:2]: yes cann

Comment (Schrodinger-09) [reply README.md:2]: so like im doing horror

**Comment by Schrodinger-09** [comment]
nihaoooo

**Comment by Schrodinger-09** [comment]
merge liao

**Comment by dawnyy** [comment]
okii 
im writing seer and priestess u who are u writing? yuesefu and malijiejie ma

**Comment by Schrodinger-09** [comment]
so u know how white sand street asylum is quite unique so i was thinking about throwing in the lydia will stand by lisa

**Comment by Schrodinger-09** [comment]
yue yue he ma li ke yi eiiiiiiii

**Comment by Schrodinger-09** [comment]
aaaaaaaaaaaaa xue she cp wo tai ke le aaaaaaa

**Comment by Schrodinger-09** [comment]
or actually y not prisoner and tracy

**Comment by Schrodinger-09** [comment]
qiu ji cp wo you ke le aaaaaaaaaaaaaaaaaa

**Comment by dawnyy** [comment]
> or actually y not prisoner and tracy

eh actually can eh big brien squad

**Pull request merged**